support/oidc-discovery-provider/config.go

```diff
@@ -26,6 +26,7 @@ type Config struct {
 	// Domain is the domain this provider will be hosted under. It is used
 	// when obtaining certs via ACME (unless ListenSocketPath is specified).
 	// Deprecated. Domains should be used instead.
+	// Deprecated: remove in 1.2.0
```

Comment:
Hmm, not sure how `domains` didn't make it in. It was in the original PR at some point. I'll add it.

As far as moving the warning, that would take a little bit of refactoring since the logger isn't initialized until after the config has been parsed. Mind if we skip that bit?

Reply:
> As far as moving the warning, that would take a little bit of refactoring since the logger isn't initialized until after the config has been parsed. Mind if we skip that bit?

oops ... yes please skip it! it will be gone in due time.